tasks/features.rake

```diff
@@ -6,10 +6,12 @@ namespace :features do
     invoke_task "test:func:all"
     invoke_task "feature:users"
     invoke_task "feature:images"
+    invoke_task "feature:barclamps"
   end
 
   desc "Run barclamp tests"
   task :barclamps do
-
+    invoke_task "feature_barclamp:ceph"
+    invoke_task "feature:barclamp:database"
```

Comment:
@vuntz: Thanks for observation. actually, its the ceph line needing correction. just corrected the same.